### PR TITLE
Add doCapture operation for ExpressCheckout

### PR DESCRIPTION
**Pull request by JavierMartinz**

### CODE REVIEW

**Comment by srmklive** [comment]
@JavierMartinz Thanks for the awesome work. Can you also add functionality for `DoAuthorization` & `DoVoid` too?

**Comment by JavierMartinz** [comment]
Sure, I would prefer do it in another PR though :)

File: src/Services/ExpressCheckout.php [hunk 1]
```diff
@@ -138,6 +138,27 @@ public function doExpressCheckoutPayment($data, $token, $payerid)
...
+    /**
+     * Function to perform DoCapture PayPal API operation.
+     *
+     * @param  string $authorization_id Transaction ID
+     * @param  float  $amount           Amount to capture
+     * @param  string $complete         Indicates whether or not this is the last capture.
+     * @param  array  $data             Optional request fields
+     * @return array
+     */
+    public function doCapture($authorization_id, $amount, $complete = 'Complete', $data = [])
```
Comment (srmklive) [inline src/Services/ExpressCheckout.php]: I would remove the `$data` parameter here. To me, it is not needed here.

File: src/Services/ExpressCheckout.php [hunk 2]
```diff
@@ -138,6 +138,27 @@ public function doExpressCheckoutPayment($data, $token, $payerid)
...
+    {
+        $response = $this->doPayPalRequest('DoCapture', array_merge($data, [
+            'AUTHORIZATIONID' => $authorization_id,
+            'AMT'             => $amount,
+            'COMPLETETYPE'    => $complete,
+            'CURRENCYCODE'    => $this->currency,
+        ]));
+
```
Comment (srmklive) [inline src/Services/ExpressCheckout.php]: As per my last comment, this should be changed to the following then:

        $response = $this->doPayPalRequest('DoCapture', [
             'AUTHORIZATIONID' => $authorization_id,
             'AMT'             => $amount,
             'COMPLETETYPE'    => $complete,
             'CURRENCYCODE'    => $this->currency,
         ]);

**Comment by srmklive** [comment]
@JavierMartinz If you can do the changes i requested. I will merge the PR.

File: src/Services/ExpressCheckout.php (hunk 1 above)
Comment (JavierMartinz) [inline src/Services/ExpressCheckout.php]: What if you would like to add a `NOTE`, for example, or any other optional parameter? https://developer.paypal.com/docs/classic/api/merchant/DoCapture_API_Operation_NVP/

Comment (srmklive) [inline src/Services/ExpressCheckout.php]: Yeah that will be better.

**Pull request merged**